app_dart/bin/server.dart

```diff
@@ -79,6 +80,11 @@ Future<void> main() async {
         config,
         authProvider,
       ),
+      '/api/reset-luci-task': ResetLuciTask(
+        config,
+        authProvider,
+        luciBuildService,
+      ),
```

Comment:
What's the difference between `luci-task` and `try-task`? Consider rename `luci-task` to `prod-task`?

Reply:
Updated to prod-task